buildscripts/parquet-reader-conda-recipe/meta.yaml

```
@@ -14,11 +14,11 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake >=3.2
     - python 3.6.*
-    - pyarrow ==0.14.1
-    - arrow-cpp ==0.14.1
+    - pyarrow ==0.15.0
```

It is possible to use [Jinja2 template](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#templating-with-jinja).
Define in begin of the file:
```
{% set PYARROW_VERSION = ">=0.14.0,<0.15.0a0" %}
```
```suggestion
    - pyarrow {{ PYARROW_VERSION }}
```
Is it ok?

---

@PokhodenkoSA yes